packages/plugin-calendar/test/integration/spec/calendar.js

```diff
@@ -0,0 +1,143 @@
+/**!
+ *
+ * Copyright (c) 2015-2016 Cisco Systems, Inc. See LICENSE file.
+ */
+
+ /* eslint camelcase: [0] */
+
+import '../..';
+import CiscoSpark from '@ciscospark/spark-core';
+import {assert} from '@ciscospark/test-helper-chai';
+import testUsers from '@ciscospark/test-helper-test-users';
+import uuid from 'uuid';
+
+const moment = require(`moment`);
+
+describe(`plugin-calendar`, function() {
+  describe(`#meetings()`, () => {
+
+    let createdMeeting, mccoy, organizer, spark, spock;
+
+    before(`create test users`, () => testUsers.create({
+      count: 3,
+      config: {
+        entitlements: [
+          `spark`,
+          `squaredCallInitiation`,
+          `squaredInviter`,
+          `squaredRoomModeration`,
+          `webExSquared`,
+          `squaredFusionCal`
+        ]
+      }
+    })
+      .then((users) => {
+        [organizer, spock, mccoy] = users;
+
+        spark = new CiscoSpark({
+          credentials: {
+            authorization: organizer.token
+          }
+        });
+      }));
+
+    const fromInterval = moment.utc().startOf(`day`).subtract(1, `week`).format(`YYYY-MM-DD[T]HH:mm:ss.SSS[Z]`);
+    const toInterval = moment.utc().startOf(`day`).add(3, `week`).format(`YYYY-MM-DD[T]HH:mm:ss.SSS[Z]`);
+
+    const startInterval = moment.utc().add(2, `hours`).format(`YYYY-MM-DD[T]HH:mm:ss.SSS[Z]`);
+    const iCalUidUUID = uuid.v4();
+    const locusId = uuid.v4();
+
+    before(`creates a meeting`, () => {
+      const meetingParams = {
+        changeType: `CREATE`,
+        userId: organizer.emailAddress,
+        userUUID: organizer.id,
+        emailAddress: organizer.emailAddress,
+        itemId: (`Item_ID-`).concat(iCalUidUUID),
+        meeting: {
+          attachments: [],
+          body: `Test Agenda`,
+          callUri: (`https://locus-example.com/locus/api/v1/loci/`).concat(locusId),
+          encryptionV2: true,
+          endDate: moment.utc(startInterval).add(1, `hours`).format(`YYYY-MM-DD[T]HH:mm:ss.SSS[Z]`),
+          featureTogglesMap: [
+            {
+              calsvc_calendar_view: true
+            }
+          ],
+          hasAttachments: false,
+          iCalUid: iCalUidUUID,
+          isRecurring: false,
+          isReminderSet: true,
+          itemId: (`Item_ID-`).concat(iCalUidUUID),
```

Comment:
i think you're generating this string in multiple locations. use variables instead. also, don't use String#concat. Instead:

```javascript
`Item_ID-${iCalUidUUID}`
```

is uiduuid really a proper identifier? And why the underscore in `Item_ID`? That seems non-standard for most of the rest of our API